src/dct.jl

```diff
@@ -3,14 +3,14 @@
 # (This is part of the FFTW module.)
 
 """
-    plan_dct!(A [, dims [, flags [, timelimit]]])
+    plan_dct!(A [, dims [, flags [, timelimit [, num_threads]]]])
```

Comment:
Why aren't these keyword arguments?

Reply:
I was just following the existing style for the keyword arguments `flags` and `timelimit`. Happy to change the documentation style (and probably flags and timelimit too?) if you'd prefer.